dir.props

```diff
@@ -69,12 +69,19 @@
     <DnuSourceList Include="https:%2F%2Fwww.nuget.org/api/v2/" />
   </ItemGroup>
 
+    <!-- list of directories to perform batch restore -->
+  <ItemGroup>
+    <DnuRestoreDir Include="&quot;$(MSBuildProjectDirectory)\src&quot;" />
```

Comment:
@jhendrixMSFT, @JeremyKuhne, I see that it's already a '\', but just to double-check, weren't we switching to '/' to better support Unix?  Or does it not matter?